plutus-benchmark/lists/bench/Bench.hs

```diff
@@ -44,11 +44,11 @@ benchmarks ctx =
       mkBMsForSort name f =
         bgroup name $ sizesForSort <&> \n ->
           bench (show n) $ benchTermCek ctx (f n)
-      sizesForSort = [10, 20..500]
+      sizesForSort = [50, 100..300]
```

Comment:
This seems like a good idea.  I think the `lists` benchmarks take a very long time (I'll run them to check), so reducing the number of cases might speed things up significantly.